Planner/Planner/src/Main.java

```diff
@@ -0,0 +1,17 @@
+import java.awt.EventQueue;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		EventQueue.invokeLater(new Runnable() {
+			@Override
+			public void run() {
+				new Users();
+				new Tasks();
```

Comment:
You can leave here just new Users(), but please rename the class Users in Planner and here call new Planner() and delete new Tasks().